src/main/java/mekanism/common/util/InventoryUtils.java

```diff
@@ -70,12 +71,21 @@ public static TransitResponse putStackInInventory(TileEntity tile, TransitReques
         return TransitResponse.EMPTY;
     }
 
+    /**
+     * Like {@link ItemHandlerHelper#canItemStacksStack(ItemStack, ItemStack)} but empty stacks mean equal (either param).
+     * Thiakil: not sure why.
```

Comment:
It is checking if two item stacks can stack with each other. From what I can gather it is more of a utils method for purposes of if a stack is empty that means we can put our stack in that slot as well, not just if they have the same item type.

Reply:
imo it shouldnt hit this method if the stack its going into  is empty

Reply:
I can see that, though I also can see how this is a nice wrapper so that you don't always have to include extra checks before hand (especially when if caring about them being empty you can directly use the `ItemHandlerHelper#canItemStacksStack` method. Was more just trying to give some clarity on why I think it is probably like this.